src/components/Dashboard.vue

```diff
@@ -337,6 +337,7 @@ export default {
       const alerts = { requirement: null, caution: null };
 
       const newCourse = {
+        crseId: course.crseId,
```

Comment:
Could we add a check here to set crseId to something if course.crseId is not defined? Not really sure what that would be though...

Reply:
A default value doesn't make sense since it will mess up with the computation